Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -945,13 +938,14 @@ class KalmanFitter {
     ACTS_VERBOSE("Preparing " << sourcelinks.size() << " input measurements");
     std::map<const Surface*, source_link_t> inputMeasurements;
     for (const auto& sl : sourcelinks) {
-      const Surface* srf = &sl.referenceSurface();
-      inputMeasurements.emplace(srf, sl);
+      inputMeasurements.emplace(&sl.referenceSurface(), sl);
```

Comment:
Any reason for this?

Reply:
Left-overs from a change that now only occurs in one of the future PRs. Restored to original implementation.